src/logic/operate.js

```diff
@@ -3,19 +3,20 @@ import Big from 'big.js';
 const operate = (numberOne, numberTwo, operation) => {
   const ONE = Big(numberOne);
   const TWO = Big(numberTwo);
-
-  if (operation === '+') {
+  if (operation === '%' && !(TWO > -1 && TWO < 1)) {
+    return ONE.mod(TWO).toString();
+  } if (operation === '%' && !(TWO > -1 && TWO < 1)) {
+    return ONE.mod(TWO).toString();
+  } if (operation === '+') {
     return ONE.plus(TWO).toString();
   } if (operation === '-') {
     return ONE.minus(TWO).toString();
-  } if (operation === '÷') {
+  } if (operation === '÷' && !(TWO > -1 && TWO < 1)) {
     return ONE.div(TWO).toString();
-  } if (operation === 'x') {
-    return ONE.times(TWO).toString;
-  } if (operation === '%') {
-    return ONE.mod(TWO).toString();
```

Comment:
- [ ]  [OPTIONAL] In fact, this operator should return percentage, not modulo. 
![1](https://user-images.githubusercontent.com/58552748/130351133-fd843b0d-05d0-40a7-a45d-ed8e32092d2f.png)